tests/modules/sightings/resources/test_create_sighting.py

```diff
@@ -129,12 +140,52 @@ def test_create_and_modify_and_delete_sighting(
         expected_status_code=400,
     )
 
+    # more complex patch: we op=remove the first encounter; should succeed no problem cuz there is one enc remaining
+    response = sighting_utils.patch_sighting(
+        flask_app_client,
+        researcher_1,
+        sighting_id,
+        patch_data=[
+            {'op': 'remove', 'path': '/encounters', 'value': enc0_id},
+        ],
+    )
+    # test to see if we now are -1 encounter
+    ct = test_utils.multi_count(db, (Sighting, Encounter, Asset, AssetGroup))
+    assert ct[1] == orig_ct[1] + 1  # previously was + 2
+
+    # similar to above, but this should fail as this is our final encounter, and thus cascade-deletes the occurrence -- and this
+    #   requires confirmation
+    response = sighting_utils.patch_sighting(
+        flask_app_client,
+        researcher_1,
+        sighting_id,
+        patch_data=[
+            {'op': 'remove', 'path': '/encounters', 'value': enc1_id},
+        ],
+        expected_status_code=400,
+    )
+    assert response.json['edm_status_code'] == 602
```

Comment:
Is this custom status code something that indicates "needs  confirmation"? I know we discussed avoiding 6xx codes unless really necessary.

Reply:
its only being used by the edm; hence it living in its own place on that response.json .... there are places edm was already using these -- but they ended up not being passable thru houston, hence that whole discussion.  they just get mapped to 400 in houston.

now i guess we could backtrack where there were 6xx codes used in the edm; it would just take some time/code changes (edm/houston); but none of those get exposed to the outside world, basically; so i just left them.